app/src/pages/inside/stepPage/stepPage.jsx

```diff
@@ -274,10 +279,24 @@ export class StepPage extends Component {
 
   handleEditDefects = (eventData) => {
     const items = eventData && eventData.id ? [eventData] : this.props.selectedItems;
-    this.props.editDefectsAction(items, {
-      fetchFunc: this.props.fetchTestItemsAction,
-      debugMode: this.props.debugMode,
-    });
+    if (
```

Comment:
I think it's better to move this condition to a new function with a descriptive name.